### DISCUSSION TITLE
Firefox ESR 128

### DISCUSSION THREAD

**Discussion by jrapf32** [Floorp-Projects/Floorp#1267] [Feature Request]
Floorp should be moved to Firefox ESR 128 due to site compatibility.

**Comment by CutterKnife** [comment]
The most recent roadmap is described in this article.
https://blog.ablaze.one/4364/2024-06-14/

**Comment by ziomek64** [comment]
Idk why ESR in the first place, now we have issue because of it

**Reply by regularentropy** [reply]
Floorp based on ESR128 will be released in this month.

**Reply by ahmed-abdelrazek** [reply]
@regularentropy any news on that?!

**Reply by CutterKnife** [reply]
See here: https://x.com/Floorp_Browser/status/1815366086172537085

**Reply by regularentropy** [reply]
> @regularentropy any news on that?!

Testing is in progress.

**Reply by ziomek64** [reply]
> > @regularentropy any news on that?!
> 
> Testing is in progress.

Fix for crashes on wayland is patched in?
